docs/content/connectors/mysql-cdc.md

```diff
@@ -463,36 +475,119 @@ Data Type Mapping
     </tr>
     <tr>
       <td>TIME [(p)]</td>
-      <td>TIME [(p)] [WITHOUT TIMEZONE]</td>
+      <td>TIME [(p)]</td>
     </tr>
     <tr>
       <td>DATETIME [(p)]</td>
-      <td>TIMESTAMP [(p)] [WITHOUT TIMEZONE]</td>
+      <td>TIMESTAMP [(p)]</td>
     </tr>
     <tr>
       <td>TIMESTAMP [(p)]</td>
-      <td>TIMESTAMP [(p)]<br>
-          TIMESTAMP [(p)] WITH LOCAL TIME ZONE
+      <td>TIMESTAMP [(p)]
+      </td>
+    </tr>
+    <tr>
+      <td>
+        CHAR(n)
+      </td>
+      <td>CHAR(n)</td>
+    </tr>
+    <tr>
+      <td>
+        VARCHAR(n)
+      </td>
+      <td>VARCHAR(n)</td>
+    </tr>
+    <tr>
+      <td>
+        BIT(n)<br>
+        where n > 1
+      </td>
+      <td>BINARY(⌊n/8⌋)</td>
+    </tr>
+    <tr>
+      <td>
+        BINARY(n)
       </td>
+      <td>BINARY(n)</td>
     </tr>
     <tr>
       <td>
-        CHAR(n)<br>
-        VARCHAR(n)<br>
-        TEXT</td>
+        VARBINARY(N)
+      </td>
+      <td>VARBINARY(N)</td>
+    </tr>
+    <tr>
+      <td>
+        TINYTEXT<br>
+        TEXT<br>
+        MEDIUMTEXT<br>
+        LONGTEXT<br>
+      </td>
       <td>STRING</td>
     </tr>
     <tr>
       <td>
-        BINARY<br>
-        VARBINARY<br>
-        BLOB</td>
+        TINYBLOB<br>
+        BLOB<br>
+        MEDIUMBLOB<br>
+        LONGBLOB<br>
+      </td>
       <td>BYTES</td>
     </tr>
+    <tr>
+      <td>
+        YEAR
+      </td>
+      <td>INT</td>
+    </tr>
+    <tr>
+      <td>
+        ENUM
+      </td>
+      <td>STRING</td>
+    </tr>
+    <tr>
+      <td>
+        JSON
+      </td>
+      <td>STRING</td>
+    </tr>
+    <tr>
+      <td>
+        SET
+      </td>
+      <td>ARRAY&lt;STRING&gt;</td>
+    </tr>
+    <tr>
+      <td>
+       GEOMETRY<br>
+       POINT<br>
+       LINESTRING<br>
+       POLYGON<br>
+       MULTIPOINT<br>
+       MULTILINESTRING<br>
+       MULTIPOLYGON<br>
+       GEOMETRYCOLLECTION<br>
+      </td>
+      <td>
+        STRING
+      </td>
+    </tr>
     </tbody>
 </table>
 </div>
 
+**Note:**
+- The precision for DECIMAL data type is up to 65 in MySQL, but the precision for DECIMAL is limited to 38 in Flink.
+  So if you define a decimal column whose precision is greater than 38, you should map it to STRING to avoid precision loss.
+- The JSON data type in MySQL will be converted into STRING with JSON format in Flink.
+- Currently, for BLOB data type in MySQL, only the blob whose length isn't greater than 2,147,483,647(2 ** 31 - 1) is supported. 
+- The spatial data types in MySQL will be converted into STRING with a fixed format like:<br>
+  ```json
+  {"srid": 0, "type": "POINT", "coordinates": [1.0,1.0]}
+  ```
+
```

Comment:
We can add a `note` column in above type mapping table, and then explain the note.

Reply:
The type mapping table is pretty long which is more than one page in the web site, and the mappings is not very clear. Could we organize it better?